website/src/pages/index.js

```diff
@@ -14,15 +14,15 @@ function Hero() {
                     <div className={clsx(styles.relative, 'row')}>
                         <div className="col">
                             <h1 className={styles.tagline}>
-                                Apify command-line interface (Apify CLI)
+                                Apify command-line interface (CLI)
                             </h1>
                         </div>
                     </div>
                     <div className="row">
                         <div className="col">
                             <h2></h2>
                             <h2>
-                                Create, develop, build, and run <a href="https://docs.apify.com/actors">Apify actors</a> from
+                                Create, develop, build, and run Apify <a href="https://docs.apify.com/actors">Actors</a> from
```

Comment:
![obrazek](https://github.com/apify/apify-cli/assets/61918049/9b3ee285-e437-48d9-a2c3-7ee23e1de6bf)

Not too sure about Apify **Actors**, I would maybe keep the link as it was - "Actors" aren't a thing (yet :)), "Apify Actors" are

Reply:
updated

Reply:
Cool, thanks! :)